manifests/v1beta1/installs/katib-openshift/kustomization.yml

```diff
@@ -0,0 +1,85 @@
+# This kustomization copies the `katib-standalone` one with following exclusions:
+#  - No Job is spawned to generate TLS key for `katib-controller` Service
+#  - Instead, the Service and WebhookConfigurations linked to it are annotated
+#    for OpenShift service controller to handle TLS certification.
+#  - PersistentVolume is not created, as OpenShift installations typically
+#    have PVC dynamical provisioning configured or need some extra administrative
+#    effort to provision a PV manually
+#  - PersistentVolumeClaim has empty `.spec.storageClassName` to allow dynamic
+#    provisioning with default StorageClass
+#
+#  Requires OpenShift version: 4.4+
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: kubeflow
+resources:
+  # Namespace.
+  - ../../components/namespace.yaml
+  # Katib controller.
+  - ../../components/controller/controller.yaml
+  - ../../components/controller/service.yaml
+  - ../../components/controller/rbac.yaml
+  - ../../components/controller/katib-config.yaml
+  - ../../components/controller/trial-templates.yaml
+  # Katib CRDs.
+  - ../../components/crd/experiment.yaml
+  - ../../components/crd/suggestion.yaml
+  - ../../components/crd/trial.yaml
+  # Katib DB manager.
+  - ../../components/db-manager/db-manager.yaml
+  - ../../components/db-manager/service.yaml
+  # Katib DB mysql.
+  - ../../components/mysql/mysql.yaml
+  - ../../components/mysql/service.yaml
+  - ../../components/mysql/pvc.yaml
+  - ../../components/mysql/secret.yaml
+  # Katib UI.
+  - ../../components/ui/ui.yaml
+  - ../../components/ui/service.yaml
+  - ../../components/ui/rbac.yaml
+  # Katib webhooks.
+  - ../../components/webhook/webhooks.yaml
+images:
+  - name: docker.io/kubeflowkatib/katib-controller
+    newTag: latest
+    newName: docker.io/kubeflowkatib/katib-controller
+  - name: docker.io/kubeflowkatib/katib-db-manager
+    newTag: latest
+    newName: docker.io/kubeflowkatib/katib-db-manager
+  - name: docker.io/kubeflowkatib/katib-ui
+    newTag: latest
+    newName: docker.io/kubeflowkatib/katib-ui
```

Comment:
The cert-generator image is missing here which might be good to add. The manifests now also have the `newTag` under the `newName` to be more inline with the manifests in the other repositories. This is the `images` from katib-standalone:

```
images:
  - name: docker.io/kubeflowkatib/katib-controller
    newName: docker.io/kubeflowkatib/katib-controller
    newTag: latest
  - name: docker.io/kubeflowkatib/katib-db-manager
    newName: docker.io/kubeflowkatib/katib-db-manager
    newTag: latest
  - name: docker.io/kubeflowkatib/katib-ui
    newName: docker.io/kubeflowkatib/katib-ui
    newTag: latest
  - name: docker.io/kubeflowkatib/cert-generator
    newName: docker.io/kubeflowkatib/cert-generator
    newTag: latest
```

Reply:
cert-generator is not needed here

Reply:
Then I will update my recommendation above.